ple/games/donkeykong/board.py

```diff
@@ -0,0 +1,350 @@
+__author__ = 'Batchu Vishal'
+import pygame
+import math
+import sys
+import os
+
+from person import Person
+from onBoard import OnBoard
+from coin import Coin
+from player import Player
+from fireball import Fireball
+from donkeyKongPerson import DonkeyKongPerson
+
+'''
+This class defines our gameboard.
+A gameboard contains everthing related to our game on it like our characters, walls, ladders, coins etc
+The generation of the level also happens in this class.
+'''
+
+
+class Board:
+    def __init__(self, width, height, rewards, rng, dir):
+        self.__width = width
+        self.__actHeight = height
+        self.__height = self.__actHeight + 10
+        self.score = 0
+        self.rng = rng
+        self.rewards = rewards
+        self.cycles = 0  # For the characters animation
+        self.direction = 0
+        self._dir = dir
+
+        self.IMAGES = {
+        	"background": pygame.image.load(os.path.join(dir, 'assets/background.png')).convert_alpha(),
```

Comment:
if the file does not use alpha, like the background call convert() instead of convert_alpha(). convert_alpha() should only be used it there is transparency in the image